docs/reference/setup-tools/kubeadm/kubeadm-upgrade.md

```diff
@@ -15,12 +14,10 @@ Every upgrade process might be a bit different, so we've documented each minor u
 Please check these documents out for more detailed how-to-upgrade guidance:
 
  * [1.6 to 1.7 upgrades](/docs/tasks/administer-cluster/kubeadm-upgrade-1-7/)
- * [1.7.x to 1.7.y  upgrades](/docs/tasks/administer-cluster/kubeadm-upgrade-1-8/)
+ * [1.7.x to 1.7.y upgrades](/docs/tasks/administer-cluster/kubeadm-upgrade-1-8/)
```

Comment:
removing the space is fine, but you also removed the {% capture overview %} tags, which are crtitical to making this template structure work. Check out this page in the [netlify deployment for this PR](https://deploy-preview-7016--kubernetes-io-master-staging.netlify.com/docs/reference/setup-tools/kubeadm/kubeadm-upgrade/) and you'll see the errors that creates

Reply:
@heckj I found that the sentences prefixed with * are all hidden in the overview section for reason I still can not find out at present, so I just move them into the body section and seems it works fine. Is this feasible?

Reply:
I just made these commits to verify why the lines are all hidden in the overview section, but still can not find out the reason. For details you can see [kubeadm upgrade](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-upgrade/), such as the [1.6 to 1.7 upgrade] lines are all hidden.

Reply:
Yes, good idea @tianshapjq - I would recommend moving them into the body if they're not showing up within the overview structure